versions/draft-2/parsers/antlr4/WdlDraft2Parser.g4

```diff
@@ -118,12 +118,17 @@ expr_infix5
   : expr_core
   ;
 
+object_literal_key
```

Comment:
AFAIK Object literals cannot accept a string as a key?

Reply:
https://github.com/openwdl/wdl/blob/main/versions/development/SPEC.md#expressions

Reply:
Oh sorry, you're specifically referring to draft-2. You're right - there is no mention of string keys for objects until v2.

Reply:
This is really confusing, because later in the Spec there is this statement: https://github.com/openwdl/wdl/blob/main/versions/development/SPEC.md#object-literals

Its also worth noting that in the development version, object literals are removed entirely in favor of struct literals. And Strucat literals do not allow for strings in the place of the key

Reply:
@jdidion

Reply:
I agree it is very confusing, but will be resolved as soon as the PR that removes object literals is merged. We can hold off on merging this PR until that one is merged, and then just remove object literals from the grammar entirely.